.changeset/thick-chicken-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/api': patch
```

Comment:
While we can consider it a bug fix, it is still a pretty significant change to the API. Thus, we should add a notice, so the change is better visible in the release notes
```md
::: notice
...
:::
```

Reply:
But we agree, that we should fix it, since it really is unintended behavior in the first place, right?

Reply:
Definitely 👌

Reply:
> @paescuj How would you suggest the notice should be worded?

Added as suggestion 👍